jenkins-pipelines/cdcReplicationPipeline.jenkinsfile

```diff
@@ -0,0 +1,180 @@
+#!groovy
+
+// Import shared libraries
```

Comment:
I think, better to use prepared the pipeline longevity and define jenkins file as for other jobs
And define the type and version of oracle in config yaml

Reply:
Unfortunately, the existing pipelines don't suit my needs :(
- I want the test to keep the cluster if any failure happens so that I can ssh into it and investigate. The longevity pipeline only keeps the cluster on internal framework failures (that are reported "critical") AFAIK.
- I want to be able to specify the oracle version from Jenkins. I'd like to have a bunch of jobs running regularly for different Scylla versions.
- I also have a patch prepared that extends this pipeline by adding an option to use gemini instead of cassandra-stress. I decided to push it as a separate PR if that one is merged.

Reply:
Longevity pipeline supports keeping the instances, you can set the `post_behavior_db_nodes` parameter:
![image](https://user-images.githubusercontent.com/5269241/87331379-8817dc80-c542-11ea-9334-bf3bafaa5b8e.png)
Regarding other things, we indeed not support since we don't have generalized parameters function

Reply:
AFAIK `keep` always keeps the instances. I want to keep them only if the test fails.
So I would like to use `keep-on-failure`, however it doesn't do exactly what I want (https://github.com/scylladb/scylla-cluster-tests/issues/1825).